src/main/java/mate/spring/into/mapper/BookMapper.java

```diff
@@ -0,0 +1,34 @@
+package mate.spring.into.mapper;
+
+import mate.spring.into.dto.BookDto;
+import mate.spring.into.dto.CreateBookRequestDto;
+import mate.spring.into.model.Book;
+import org.springframework.stereotype.Component;
+
+@Component
+public class BookMapper {
```

Comment:
![image](https://github.com/dimonkoval/mate-springboot-into/assets/164332253/25b1a20c-ed34-4407-8cf4-9683923a4413)
Please use MapStruct instead of you own Mapper